### DISCUSSION TITLE
Notifications for memories from PhotoPrism

### DISCUSSION THREAD

**Discussion by carlonluca** [photoprism/photoprism#4119] [Show and tell]
Hello,
I was interested in the feature requested in this issue: https://github.com/photoprism/photoprism/issues/720.

While waiting for an official implementation, I wrote a quick alternative approach as a (partially) cross-platform app here: https://github.com/carlonluca/flashbackprism. It provides notifications when photos are available and allows to browse by year. A few users are using it on Android already. Linux, Windows and Mac OS are also working.

More info here: https://bugfreeblog.duckdns.org/2024/01/revisit-todays-photos-from-n-years-ago-with-photoprism.html.

I do not own a Google account to share it, so I only uploaded arm64 packages to GitHub.

For those interested, this is the repo: https://github.com/carlonluca/flashbackprism. App is 100% free and open source.

Best regards.

**Comment by graciousgrey** [comment]
Looks great :)

If you like, you're very welcome to create a pull request to list your app here:

https://github.com/photoprism/photoprism-contrib

**Reply by carlonluca** [reply]
Why not? I'll try! Thanks!
